### DISCUSSION TITLE
Documentation incubation inside ocaml.org

### DISCUSSION THREAD

**Discussion by cuihtlauac** [ocaml/ocaml.org#1290] [General]
Right now, reviewing updated or new documentation requires either
- reading the markdown source in GH
- setting up a build environment to see the how the new doc looks like

It is possible to push to the staging branch, but it requires git rebase which can be painful.

It should be easier. 

One way to do that would be to publish new docs inside ocaml.org. Options would be:

* Have the new docs mixed with the exiting docs, but labelled as beta or work-in-progress
* Have the new docs in a dedicated area (à la Area 51) 

Extra requirements

* Quick and easy liaison with corresponding github PR
* It must be possible to have two versions of the same documentation, that's required for update of existing documents

**Comment by christinerose** [comment]
When I review them, I generally read the Markdown in GH along with looking at the published piece, but of course that only works for ones published! 

While I don't have to do the build environment preview part, that sounds quite annoying. Does it go through Jekyll or Gatsby or something?

I love the idea of having a "beta" or "WIP" caveat at the top, perhaps even asking for community feedback.
